ansible/roles/sonicv2/tasks/sonic-cavm.yml

```diff
@@ -0,0 +1,58 @@
+# SONiC
+
+# Setup Platform
+#- include: platform-cavm.yml
+
+# Install Cavium Host Interface Driver
+- block:
+  - name: Rmmod for driver
```

Comment:
modprobe should be integrated into xp80PcieEndpoint deb packet, pre-install script